test/app/components/Settings/ConnectButton/ConnectButton.test.js

```diff
@@ -0,0 +1,289 @@
+jest.dontMock('../../../../../app/components/Settings/ConnectButton/ConnectButton.react.js');
```

Comment:
Are we affected by [this issue](https://facebook.github.io/jest/docs/en/troubleshooting.html#i-m-using-babel-and-my-unmocked-imports-aren-t-working) with unmocked imports?

Reply:
No.  Jest mocks modules by default.  I specify that it should not mock this component so that we can test the component.  This is a pattern that I have used before on other projects and seems to have worked

Reply:
I understand. What I meant is whether we need to use `unmock` instead.

Reply:
@n-riesco It looks like unmock is now the recommended approach.  Here is a link to a link on SO (https://stackoverflow.com/questions/36571357/difference-between-unmock-and-dontmock-in-jest).  I have updated the test and and added it to the PR